tests/integration/testdata/publish/template_create_app.yaml

```diff
@@ -0,0 +1,38 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Description: 'sam-app
+
+  Sample SAM Template for sam-app
+
+  '
+Globals:
+  Function:
+    Timeout: 3
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Author: user
+    Description: description
+    HomePageUrl: https://github.com/test/test
+    Labels:
+    - test-app
+    LicenseUrl: s3://test-sam-publish-app/LICENSE.txt
```

Comment:
we want the integration tests to work without files being already present in S3, right? Anyone should be able to run the integration tests, can we Ref a local one or upload new files to s3 during integ test and then parameter over-ride them.

Reply:
Thanks for the suggestion! I refactored the tests to run without assuming S3 file existence.